index.html

```diff
@@ -14,6 +14,73 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
-</body>
+
+<div class="container">
+    <header>
+        <nav>
+          <a href="#">Home</a>
+          <a href="#">About</a>
+          <a href="#">Portfolio</a>
+          <a href="#">Contact</a>
+        </nav>
+    </header>
+
+  <div class = "sectionone">
+    <img class = "vader" src="https://geeklash.files.wordpress.com/2014/03/mr-vader-e1395860584515.jpg?w=476&h=600">
+      <div class = "one">
+        <h1>Darth Vader - Aspiring Web Developer</h1>
+    <p>Retired Sith Lord seeking to utilize my natural force powers to launch a new career in web development.</p>
+        </div>
+  </div>
+
+  <div class = "sectiontwo">
+    <div class = "sum">
+      <h1>Summary</h1>
+      <p>It is a period of civil war. Rebel spaceships, striking from a hidden base, have won their first victory against the evil Galactic Empire. During the battle, Rebel spies managed to steal secret plans to the Empire’s ultimate weapon, the DEATH STAR, an armored space station with enough power to destroy an entire planet. Pursued by the Empire’s sinister agents, Princess Leia races home aboard her starship, custodian of the stolen plans that can save her people and restore freedom to the galaxy….</p>
+    </div>
+    <div class="strenths">
+      <h1>Strenths</h1>
+        <div class = "list">          
+            <p>High midichlorian count</p>
+            <p>Force choking</p>
+            <p>Moves stuff with my mind</p>
+            <p>Cool mask</p>            
+        </div>
+      </div>
+    </div>
+
+  <div class = "history">
+    <h1>Work History</h1>
+    </div>
+  
+  <div class = "stuff">
+      <div class = "year">
+        <h2>Years</h2>
+        <p>10</p>
+        <p>5</p>
+        <p>10</p>
+      </div> 
+      <div class = "company">
+        <h2>Company</h2>
+        <p>Watto</p>
+        <p>Jedi Academy</p>
+        <p>Emprire</p>
+      </div>
+      <div class = "tle">
+        <h2>Title</h2>
+        <p>Slave</p>
+        <p>Jedi Knight</p>
+        <p>Sith Lord</p>
+      </div>
+      <div class = "Reason">
+        <h2>Reason For Leaving</h2>
+        <p>Freed from slavery</p>
+        <p>Turned Evil</p>
+        <p>Died</p>
+      </div>
+
+    </div>
```

Comment:
Nice simple design here. If you were curious what was used in the design file, take a look at [HTML Tables](https://html.com/tables/)!